pkg/compiler/types/spec/spec.go

```diff
@@ -242,7 +242,7 @@ func (cs *LuetCompilationSpec) SetSeedImage(s string) {
 }
 
 func (cs *LuetCompilationSpec) EmptyPackage() bool {
-	return len(cs.BuildSteps()) == 0 && len(cs.GetPreBuildSteps()) == 0 && !cs.UnpackedPackage()
+	return len(cs.BuildSteps()) == 0 && !cs.UnpackedPackage()
```

Comment:
In this case you should also check if `package_dir` is not set

Reply:
mmm is not handled by UnpackedPackage()?

Reply:
I confirm: https://github.com/mudler/luet/blob/6305d6a556a797eb782773f1ef79cc9be55068a4/pkg/compiler/types/spec/spec.go#L248

the UnpackedPackage() return true if package_dir is defined.